ftl-project.toml

```diff
@@ -1,8 +1,6 @@
 name = "ftl"
 module-dirs = ["examples/go"]
 ftl-min-version = ""
-hermit = false
```

Comment:
Why'd these get removed?

Reply:
oh what the heck! I'll undo all the changes to that file. Thanks for catching that